include/deal.II/matrix_free/shape_info.templates.h

```diff
@@ -77,6 +77,96 @@ namespace internal
     }
 
 
+    template <int dim>
+    void
+    get_element_type_specific_information(
+      const FiniteElement<dim, dim> &fe_in,
+      const FiniteElement<dim, dim> *fe,
+      const unsigned int             base_element_number,
+      ElementType &                  element_type,
+      std::vector<unsigned int> &    scalar_lexicographic,
+      std::vector<unsigned int> &    lexicographic_numbering)
```

Comment:
@peterrum  I've been looking at this patch for a little (see #11313). Can you add some documentation to this function? It's not clear from the naming of the arguments what the first two arguments represent -- `fe` and `fe_in` doesn't give away much as to how these two finite elements related to each other.